modules/structured_light/src/slmono.cpp

```diff
@@ -0,0 +1,242 @@
+#include <opencv2/structured_light/slmono.hpp>
```

Comment:
Please use short OpenCV license header (in all new .hpp/.cpp files except samples):
```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```
https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure